raiden/raiden_service.py

```diff
@@ -194,6 +194,7 @@ def initiator_init(
     token_network_address: TokenNetworkAddress,
     target_address: TargetAddress,
     lock_timeout: BlockTimeout = None,
+    routes: List[RouteState] = None,
```

Comment:
Nitpick: It might make sense to call this and the occurrences below `route_states` to prevent confusion.